catalog/static/json-editor.css

```diff
@@ -0,0 +1,3 @@
+.jsoneditor.jsoneditor {
+    border: 0;
+}
```

Comment:
pls add a trailing newline for consistency

Reply:
Trying to live with VS Code :)